source/help/getting-started/managing-members.rst

```diff
@@ -52,8 +52,8 @@ Member
 
 This is the default role given to users when they join a team. Members have basic permissions on the Mattermost team.
 
-Channel Admin
-~~~~~~~~~~~~~
+Channel Admin (Enterprise Edition only)
```

Comment:
Please look at the config settings doc for how we typically format things that are enterprise only